middleware/csrf.go

```diff
@@ -124,7 +124,7 @@ func CSRFWithConfig(config CSRFConfig) echo.MiddlewareFunc {
 
 			req := c.Request()
 			k, err := c.Cookie(config.CookieName)
-			token := ""
+			var token string
```

Comment:
As above

Reply:
ineffectual assignment

Reply:
How about?

```go
token := k.Value
if err != nil {
    token = random.String(config.TokenLength)
}
```

We have both short variable declaration, avoid the ineffectual assignment and ditch the else clause.